manpages/windows/v1.3/rpmemd.1.md

```diff
@@ -0,0 +1,235 @@
+---
```

Comment:
Ha! The file is there, but it's "invisible" as there is no link to it on pmem.io. Same situation is on master. 
I'll remove the unneeded files.